doc/mvf_v4.rst

```diff
@@ -225,36 +225,42 @@ Calibration solutions
 
 Streams of type ``sdp.cal`` have the following keys.
 
-``antlist`` (list of string)
+``antlist`` (list of string, length ``n_ants``)
     List of antenna names. Arrays of calibration solutions use this
     order along the antenna axis.
 
-``pol_ordering`` (list of string)
+``pol_ordering`` (list of string, length ``n_pols``)
     List of polarisations (from ``v`` and ``h``). Arrays of calibration
     solutions use this order along the polarisation axis.
 
-``bls_ordering`` (2D array)
+``bls_ordering`` (list of string pairs, length ``n_bls``)
     Same meaning as for ``sdp.vis`` streams, but describes the internal
     ordering used within the calibration pipeline and not of much use to
     users.
 
 ``param_*``
     Parameters used to configure the calibration.
 
-``product_G`` (2D array) — sensor
-    Gain solutions (derived e.g. on a phase calibrator), indexed by antenna
-    and polarisation. The complex values in the array apply to the entire band.
+``refant`` (string)
+    Name of the selected reference antenna (which will also appear in
+    ``antlist``). The reference antenna is only chosen when first needed in
+    a capture block, so this key may be absent if there was no calibration yet.
```

Comment:
Need to document that older files will have a description string instead of just a name here.

Reply:
Done.